docs/plugins.md

```diff
@@ -1525,6 +1526,17 @@ These plugins provide new markers or news ways of converting abstract data into
 			<a href="https://github.com/Oliv">Olivier Gasc</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/JackZouShao/leaflet-marker-drection">leaflet-marker-drection</a>
+		</td>
+		<td>
+			display the path and the direction of the marker. (<a href='https://jackzoushao.github.io/leaflet-marker-drection/examples/marker-drection.html/'>demo</a>)
```

Comment:
The link does not work (404 error). Most probably the trailing slash (`/`) should be removed.

Reply:
OK.I have removed this trailing slash .

Reply:
Do I need to pull a new request?

Reply:
no need for a new PR, just remove `<te` line, as mentioned above

Reply:
I just removed <te line at line NO.1045